.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,18 @@
+# This workflow will launch at 00:00 every Sunday
+# This file is based on docs from https://github.com/scala-steward-org/scala-steward-action
+on:
+  schedule:
+    - cron: '0 12 * * *'
+
+name: Scala Steward CI
+
+jobs:
+  scala-steward:
+    runs-on: ubuntu-20.04
+    name: Launch Scala Steward
```

Comment:
Should we add an `if` condition so that this is only triggered on the `apache/daffodil-vscode` repo? If someone creates a fork of the repo we probably don't want their fork to be spammed with PR's updating dependencies. Dependabot does that right now with no way to disable it and I find it pretty annoying.

I think something like would work:
```yaml
jobs:
  scala-steward:
    if: github.repository == 'apache/daffodil-vscode'
    ...
```

Reply:
Makes sense to me I added this in